docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/yabloc-guide.md

```diff
@@ -0,0 +1,97 @@
+# Using YabLoc in Autoware
```

Comment:
It's rather a broad suggestion which is pretty much open to discussion, but since it seems that the contents are mostly duplicated with the Readme.md of YabLoc in Autoware Universe. I guess it's better to avoid maintaining such duplicated documents, so how about removing this document and merge it into Readme, or the other way around (Remove readme and just put a link to this document)?

Reply:
Thank you for the suggestion. It's a great idea.
Since the "limitations" and "input/output" are already written in the README of `autoware.universe`, I will try to reconsider the content of this page to avoid duplication. :raising_hand_man: